src/app/app.component.html

```diff
@@ -1,12 +1,14 @@
-<header>
-  <h1>Parallax effect</h1>
-  <p>Pure CSS. No JS</p>
-  <span>
-    <p>This is my first working session in VS Code.</p>
-    <p>Also I Use GitBusher for virtual server creation,<br>
-    and GitKraken for version control and changes commitment.</p>
-  </span>
-</header>
-<div class="logo">
+<div class="body-block">
```

Comment:
Did You add .body-block as a wrapper for all page content?

Reply:
Yes, just to make sure that it will be rendered as a one tag at first